test/parallel/test-process-clobber.js

```diff
@@ -0,0 +1,5 @@
+/* eslint-disable no-global-assign */
+/* eslint-disable required-modules */
+'use strict';
+
```

Comment:
I added the required-modules eslint directive so I didn't have to include test/common.js, because while core is now robust to tampering, the code in test/common.js need not be.